### ISSUE TITLE
Translation / weblate

### ISSUE THREAD

**Issue by Babag63** [Nitrokey/nitrokey-documentation#369]
Hi, I was trying to follow documentations in French and let's say it's not always perfect.
I've seen you have been using DeepL for the first ones so it explains why, but you have already set up (it seems) weblate.

So my question is, could you make your translations available to some people on Weblate ? Maybe not everybody, so you can be reassured about vandalism and that kind of things, but right now I'd like to help with translations and I haven't seen any option to do so (French in my case).
Should I do in a first time a PR for some files like [these](https://github.com/Nitrokey/nitrokey-documentation-data/tree/main/locales/fr/LC_MESSAGES) ?

If I just haven't seen the instructions, could you write them to the README of nitrokey-documentation-data or nitrokey-documentation (or both) ?

Finally, the "Edit this page" button doesn't work on some pages not in English, see [this one](https://docs.nitrokey.com/fr/nitrokeys/) for example.

**Comment by jans23** [comment]
In deed it would be great to get your and other contributions on translations and we should document this too. Please register an account at translate.nitrokey.com and let me know if you need additional permissions in order to work on the translation.

No PRs for translations, please.

**Comment by Babag63** [comment]
I did and it says I'm in the Users ans Viewers group but I can't create any project or see existants.
I'm @baba in weblate.

**Comment by jans23** [comment]
Now I granted you User permissions. Can you see and work on the project now?

**Comment by Babag63** [comment]
It seems not to be enough as when clicking [on the french project](https://translate.nitrokey.com/projects/nitrokey-documentation/root/fr/) that the weblate bot links to [on the last commit](https://github.com/Nitrokey/nitrokey-documentation-data/commit/eda9ae7dd4796560ade8e6fdbea03df10b994117), it tells me that the project must not exist or must not be publicly available.
And I can't see anything in the [global projects page](https://translate.nitrokey.com/projects/).
Maybe am I looking at the wrong places ?

**Comment by daringer** [comment]
> Finally, the "Edit this page" button doesn't work on some pages not in English, see [this one](https://docs.nitrokey.com/fr/nitrokeys/) for example.

yes, "edit this page" links inside the translation are still incorrect

> It seems not to be enough as when clicking [on the french project](https://translate.nitrokey.com/projects/nitrokey-documentation/root/fr/) that the weblate

this should be the right spot, and for me it also shows content, I'd guess there is something not correct with the permissions, we'll check

**Comment by Babag63** [comment]
I now have access to all languages!
I'm not super familiar with weblate but I now can edit a looot of things seemingly without need to approval: I'll be cautionnous don't worry, it's just that this level of access may not (or may, idk) be suitable for extending the number of people able to do so.

Also some translations are "locked" like [this one](https://translate.nitrokey.com/translate/nitrokey-documentation/components/fr/?checksum=06866ca19e28a1d6), and [this](https://translate.nitrokey.com/projects/nitrokey-documentation/components/#alerts) page explains that it's because of an alert but you may already know.

**Comment by daringer** [comment]
> Also some translations are "locked" like [this one](https://translate.nitrokey.com/translate/nitrokey-documentation/components/fr/?checksum=06866ca19e28a1d6), and [this](https://translate.nitrokey.com/projects/nitrokey-documentation/components/#alerts) page explains that it's because of an alert but you may already know.

yes, this is "a feature", i.e., just the old translation which was not yet deleted entirely, please just ignore them
